js/portofolio.js

```diff
@@ -1,7 +1,104 @@
+const dataSource = [
```

Comment:
- [ ] You are almost there! However, for some reason the projects section is hidden when the site is loaded, therefore I was not able to review it:

<img width="1437" alt="Screen Shot 2022-09-08 at 11 32 11 PM" src="https://user-images.githubusercontent.com/83717285/189272657-d78ed9b4-814c-46ba-bcf8-609a0c0596f4.png">

Let's fix that bug to get it to display this section.

Reply:
thank you 
please would you mind checking it again
i forgot a last push

Reply:
Hi @smunozmo, 
thank you
please would you mind checking it again
i forgot a last push but i just push it